src/applications/resources-and-support/components/SearchBar.jsx

```diff
@@ -46,7 +46,7 @@ export default function SearchBar({
   };
 
   return (
-    <div className="vads-u-border-top--2px vads-u-border-color--gray-light vads-u-padding-y--3 vads-u-padding-x--1">
+    <div className="vads-u-border-bottom--0px medium-screen:vads-u-border-top--2px vads-u-border-color--gray-light vads-u-padding-y--3 vads-u-padding-x--1">
```

Comment:
You want `vads-u-border-bottom--0`, without the `px`. [Reference](https://design.va.gov/utilities/border)